package.json

```diff
@@ -47,7 +47,8 @@
           "import/no-duplicates": "off"
         }
       }
-    ]
+    ],
+    "parser": "babel-eslint"
```

Comment:
Can we log an issue to eslint-config-terra to considering pulling this in to the default config?

Reply:
agreed, this is a great candidate for our default config. optional chaining for everyone :)

Reply:
Added: https://github.com/cerner/terra-toolkit/issues/415